bigchaindb/config_utils.py

```diff
@@ -97,8 +98,11 @@ def file_config(filename=None):
         filename = CONFIG_DEFAULT_PATH
 
     logger.debug('file_config() will try to open `{}`'.format(filename))
-    with open(filename) as f:
-        config = json.load(f)
+    try:
+        with open(filename) as f:
+            config = json.load(f)
+    except Exception as err:
```

Comment:
Thank you for pointing out! That's why I was asking. Just worth noting that catching `Exception` is not really an antipattern as described in the blog post you have mentioned, as it was rethrowing new type of exception with details about the location of problem, and just silently suppressing it.

However, I agree that mixing file open and JSON parsing exceptions was a bad idea, because `FileNotFound` (or any other IO exceptions?) are handled from the calling functions. 

So again, my question is, addressing the original issue, should we replace the `ValueError` raised by `json.load()` by custom exception or let calling function to handle `ValueError` as they do with `FileNotFound`?